elevation_mapping/include/elevation_mapping/ElevationMapping.hpp

```diff
@@ -175,6 +183,12 @@ class ElevationMapping
    */
   void stopMapUpdateTimer();
 
+  /*!
+   * Get a gridMapMessage of a Submap or rawSubmap
```

Comment:
The arguments are not documented for this method.

Reply:
"Get a Grid Map ROS message of a submap of the raw or fused submap."